force_wfmanager/wfmanager_setup_task.py

```diff
@@ -213,6 +221,13 @@ def _tool_bars_default(self):
                     method="open_plugins",
                     image_size=(64, 64)
                 ),
+                TaskAction(
```

Comment:
I suppose later on we could include an initialisation window at the beginning of the app to just run any custom UI outside of `WfManagerSetupTask`? Ultimately it just creates a workflow file that can be ran by the headless `force-bdss` and then analysed on `WfManagerReviewTask`.

This could allow us to hide certain complexities from the user, depending on their technical knowledge.

Reply:
Additionally, running the custom UI inside `force_wfmanager` means including `force_wfmanager` as a requirement for any plugin that contributes a custom UI (force-h2020/force-bdss-plugin-enthought-example#28). Therefore it would be even better to allow the custom UI to run as a stand alone app, as well as within the `force_wfmanager`. However, this can be achieved in later PRs.